src/state/GlobalState.tsx

```diff
@@ -103,14 +106,35 @@ export class GlobalState extends PureComponent<Props> {
     showHidden: false,
     favouriteGames:
       (configStore.get('games.favourites') as Array<FavouriteGame>) || [],
-    theme: (configStore.get('theme') as string) || ''
+    theme: (configStore.get('theme') as string) || '',
+    zoomPercent: parseInt((configStore.get('zoomPercent') as string) || '100'),
+    contentFontFamily:
+      (configStore.get('contentFontFamily') as string) || "'Cabin', sans-serif",
+    actionsFontFamily:
+      (configStore.get('actionsFontFamily') as string) || "'Rubik', sans-serif"
   }
 
   setTheme = (newThemeName: string) => {
     configStore.set('theme', newThemeName)
     this.setState({ theme: newThemeName })
   }
 
+  setZoomPercent = (newZoomPercent: number) => {
+    configStore.set('zoomPercent', newZoomPercent)
```

Comment:
Adding these changes inside a `setTimeout` with 300ms set makes the UI better to use IMO, because then it doesn't change abruptly.

Reply:
I'll try this, I noticed the weird jumps but I didn't really know what to try here, I'll check with a delay

I don't think I can transition zoom, because it's not css zoom, is chrome's zoom and it doesn't have transition (unless I implement a gradual change by setting the intermediate values in some loop but I know that would be bad for people that prefers reduced motion)

Reply:
just sent a commit with the delay, looks way better :muscle:

Reply:
NIce, so we can merge like it is now and we can improve that later. I think it is stable enough for the next release.